debian/install

```diff
@@ -1,4 +1,6 @@
 etc/
 lib/
 usr/
-var/
+var/lib/delphix-appliance/platform
+var/lib/delphix-platform/
+var/target
```

Comment:
Where is var/target used?

Reply:
see here: https://github.com/delphix/delphix-platform/commit/0fc62878f835e2f04b1d4659a1055235ea56b84e